src/lib/y2storage/blk_device.rb

```diff
@@ -82,6 +82,36 @@ def blk_filesystem
       encrypted? ? encryption.direct_blk_filesystem : direct_blk_filesystem
     end
 
+    alias_method :filesystem, :blk_filesystem
+
+    # Label of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_label
+      return nil unless blk_filesystem
+      blk_filesystem.label
+    end
+
+    # UUID of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_uuid
+      return nil unless blk_filesystem
+      blk_filesystem.uuid
+    end
+
+    # Type of the filesystem, if any
+    # @return [Filesystems::Type, nil]
+    def filesystem_type
+      return nil unless blk_filesystem
+      blk_filesystem.type
+    end
+
+    # Mount point of the filesystem, if any
+    # @return [String, nil]
+    def filesystem_mountpoint
+      return nil unless blk_filesystem
+      blk_filesystem.mountpoint
+    end
```

Comment:
or better, why these method deserve shortcut or others not? Maybe if there is some logic in it, then API make some sense like shortcuts for ...